manual.h

```diff
@@ -0,0 +1,6 @@
+void getManualFromNumber(int pairNumber)
```

Comment:
The manual should have all numbers as a reference... also, the result is expected in a big string and not console-output

Reply:
I have made changes accordingly and also renamed the files.Please check it.

Reply:
Done 👍 
Minor point, you can use the += operator when the lvalue is on the right side as well.